src/app/static/src/app/components/plots/utils.ts

```diff
@@ -173,4 +174,17 @@ export const findPath = function (id: string, obj: any): any {
           }
       }
   }
-}
\ No newline at end of file
+};
+
+export const formatOutput = function (value: number | string, form: string) {
```

Comment:
I think this function should take the indicator metadata as a parameter rather than form. Once Rob's put in the backend then the scale and format values should be on the indicator metadata, and you can use those here to first scale the value, then pass the scaled value and format to numeral. 
Until the backend is available: rather than making a dictionary of formats, I was imagining just switching on the indicator id, to decide realistic scale and format for each.